src/effects/VST/VSTEffect.cpp

```diff
@@ -1104,22 +1104,15 @@ void VSTEffectMessage::Merge(Message && src)
 
 std::unique_ptr<EffectInstance::Message> VSTEffectInstance::MakeMessage() const
 {
+   // The purpose here is just to allocate vectors (chunk and paramVector)
+   // with sufficient size, not to get the values too
    VSTEffectSettings settings;
-   FetchSettings(settings, false);
+   FetchSettings(settings, /* doFetch = */ false);
 
    VSTEffectMessage::ParamVector paramVector;
    paramVector.resize(settings.mParamsMap.size(), std::nullopt);
```

Comment:
Better simply `mAEffect->numParams` for the vector size

Reply:
ok